src/components/label.ts

```diff
@@ -19,12 +19,13 @@ export module Components {
      *
      * @constructor
      * @param {string} displayText The text of the Label (default = "").
+     * @param {number} angle The rotation angle of the text (default = 0). 
```

Comment:
Mention that 0 means horizontal. As a reader of this documentation, I have no idea what a rotation of 0 means in this library